src/EditorFeatures/CSharpTest/ConvertToInterpolatedString/ConvertConcatenationToInterpolatedStringTests.cs

```diff
@@ -399,5 +399,74 @@ void M()
     }
 }");
         }
+
+        [WorkItem(23536, "https://github.com/dotnet/roslyn/issues/23536")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertToInterpolatedString)]
+        public async Task TestWithStringLiteralWithBraces()
+        {
+            {
+                await TestInRegularAndScriptAsync(
+    @"public class C
+{
+    void M()
+    {
+        var v = 1 + [||]""{string}"";
+    }
+}",
+    @"public class C
+{
+    void M()
+    {
+        var v = $""{1}{{string}}"";
+    }
+}");
+            }
+        }
+
+        [WorkItem(23536, "https://github.com/dotnet/roslyn/issues/23536")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertToInterpolatedString)]
+        public async Task TestWithStringLiteralWithDoubleBraces()
+        {
+            {
+                await TestInRegularAndScriptAsync(
+    @"public class C
+{
+    void M()
+    {
+        var v = 1 + [||]""{{string}}"";
+    }
+}",
+    @"public class C
+{
+    void M()
+    {
+        var v = $""{1}{{{{string}}}}"";
+    }
+}");
+            }
+        }
+
+        [WorkItem(23536, "https://github.com/dotnet/roslyn/issues/23536")]
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsConvertToInterpolatedString)]
+        public async Task TestWithMultipleStringLiteralsWithBraces()
+        {
+            {
+                await TestInRegularAndScriptAsync(
+    @"public class C
+{
+    void M()
+    {
+        var v = ""{"" + 1 + [||]""}"";
+    }
+}",
+    @"public class C
+{
+    void M()
+    {
+        var v = $""{{{1}}}"";
+    }
+}");
```

Comment:
can you add tests with verbatim literals as well?  Thanks!

Reply:
Thanks for your review. I added two unit tests for verbatim literals as well.